util/ion/broadcast_test.go

```diff
@@ -29,6 +30,10 @@ type myFakeTx struct {
 	msgs []myFakeMsg
 }
 
+func (t myFakeTx) GetMsgsV2() ([]protov2.Message, error) {
+	return nil, nil // this is a hack for tests
+}
```

Comment:
Why exactly is this needed? Is this a new requirement on an interface from cosmos? If so, can you leave a link and some extended docs in the code ?

Reply:
Just for the test case we needed this function, [refer](https://github.com/cosmos/cosmos-sdk/blob/9814f684b9dd7e384064ca86876688c05e685e54/types/mempool/mempool_test.go#L79) for similar case